notes-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java

```diff
@@ -1265,7 +1266,7 @@ public Response searchData(@Context
                                      List<String> tagNames,
                                      @Parameter(description = "Space id used to search notes of one space", required = false)
                                      @QueryParam("spaceId")
-                                     String spaceId,
+                                     List<String> spaceIds,
```

Comment:
Use Long rather than String to optimize Data Type conversions

```suggestion
                                     List<Long> spaceIds,
```

Reply:
done!